pkg/grafana/provider.go

```diff
@@ -79,5 +79,6 @@ func (p *Provider) GetHandlers() []grizzly.Handler {
 		NewDashboardHandler(p),
 		NewRuleHandler(p),
 		NewSyntheticMonitoringHandler(p),
+		NewNotificationPolicyHandler(p),
```

Comment:
Same comment as in https://github.com/grafana/grizzly/pull/289
I'd call it `AlertingNotificationPolicy`

Reply:
I've renamed it to `AlertNotificationPolicy` to keep in line with `AlertRuleGroup` for now. If we want to rename this, we can do everything at once at a later point